src/app/pages/courses-page/load-more-panel/load-more-panel.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-load-more-panel',
   templateUrl: './load-more-panel.component.html',
   styleUrls: [ './load-more-panel.component.scss' ],
 })
-export class LoadMorePanelComponent implements OnInit {
-
+export class LoadMorePanelComponent {
   constructor() { }
 
-  ngOnInit(): void {
+  public loadMoreHandler() {
```

Comment:
There is no necessity to use `public` access modifier at all. all props are public by default.

Reply:
Ok